pkg/testutils/full_init.go

```diff
@@ -9,36 +9,27 @@ import (
 	"unicode"
 	"unsafe"
 
+	types "github.com/stackrox/rox/pkg/protocompat"
+	"github.com/stackrox/rox/pkg/utils"
 	"github.com/stackrox/rox/pkg/uuid"
 )
 
 // BasicTypeInitializer prescribes how to initialize a struct field with a given type.
 type BasicTypeInitializer interface {
-	Value(ty reflect.Type, fieldPath []reflect.StructField) interface{}
+	Value(ty reflect.Kind, fieldPath []reflect.StructField) interface{}
```

Comment:
nit: As `ty` is now of type `reflect.Kind` instead of `reflect.Type`, I'd rename it to `kind`.
This applies to all changed `Value` functions.